src/components/dashboard/FeedItems/EventCounterParty.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Text } from 'react-native-web'
+import { Text } from 'react-native'
 import { listStyles } from './EventStyles'
```

Comment:
According to what we discuss and put in the styling guidelines we should remove this general style file and split each style either on a reusable component or just moving the style to the component in which is being used

Reply:
Done